B827EBFFFE820C3F.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "B827EBFFFE820C3F",
+    "servers": [
+      {
+        "server_address": "eu.thethings.network",
```

Comment:
This is not the correct server address, please check the valid addresses for routers here: https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses